components/safe_browsing/android/java/src/org/chromium/components/safe_browsing/BraveSafeBrowsingApiHandler.java

```diff
@@ -24,6 +24,7 @@
 import org.chromium.base.Log;
 
 public class BraveSafeBrowsingApiHandler implements SafeBrowsingApiHandler {
+    public static final long SAFE_BROWSING_INIT_INTERVAL = 30000;
```

Comment:
nit: I'd call the variable `SAFE_BROWSING_INIT_INTERVAL_MS` just to make the unit explicit.